test/test_integration.py

```diff
@@ -1446,3 +1452,127 @@ def _insert_cookie(rep):
         self.wait_for_text_to_equal('#content', 'Logged out')
 
         self.assertFalse(self.driver.get_cookie('logout-cookie'))
+
+    def test_lose_focus_input(self):
```

Comment:
Is this test really different from the `test_simple_callback` one? There, on line 1569, the input also loses focus. Perhaps this test could be removed?